### PR TITLE
Add upvote/downvote buttons

### PR DESCRIPTION
**Pull request by ccjeane**
- Add place for upvote/downvote buttons on each review
- Delete code that has been rendered obsolete by the recent datastore changes

### CODE REVIEW

File: src/main/webapp/index-script.js [hunk 1]
```diff
@@ -444,7 +444,24 @@ function generateReview(review) {
   const reviewText = document.createElement('p');
   reviewText.innerHTML += review;
 
+  // TODO: Change innerHTML of both buttons to be something else or add an icon 
+  const upvoteButton = document.createElement('button');
+  upvoteButton.innerHTML += "Good";
```
Comment (kcirerick) [inline src/main/webapp/index-script.js:449]: innerHTML = '&#1F44D'

File: src/main/webapp/index-script.js [hunk 2]
```diff
@@ -444,7 +444,24 @@ function generateReview(review) {
...
+  upvoteButton.id += "upvote";
+  upvoteButton.addEventListener("click", () => { 
+    // TODO: Figure out what to do upon click.
+  });
+
+  const downvoteButton = document.createElement('button');
+  downvoteButton.innerHTML += "Bad";
```
Comment (kcirerick) [inline src/main/webapp/index-script.js:456]: innerHTML = '&#1F44E'

Comment (ccjeane) [reply src/main/webapp/index-script.js]: Incredible, thanks!

File: src/main/webapp/index-script.js (hunk 1 above)
Comment (ccjeane) [reply src/main/webapp/index-script.js]: Remarkable, much appreciated!

**Pull request merged**